### PR TITLE
Comics-1: comics page and navigation

### PR DESCRIPTION
**Pull request by Ellyia**

### CODE REVIEW

File: src/app/core/components/navigation/navigation.component.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import 'mixines';
```
Comment (Phantomus) [inline src/app/core/components/navigation/navigation.component.scss]: seems you do not use any mixin or variable here

File: src/app/core/enums/active-route-nav.enum.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export enum ActiveRouteNavigationEnum {
+  CharactersRoute,
```
Comment (Phantomus) [inline src/app/core/enums/active-route-nav.enum.ts]: it is not necessary to use 'Route' in each enum value

File: src/app/core/components/navigation/navigation.component.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+export class NavigationComponent implements OnDestroy {
+  activeRouteNavigationEnum = ActiveRouteNavigationEnum;
+
+  activeTab: ActiveRouteNavigationEnum = ActiveRouteNavigationEnum.CharactersRoute;
+
+  routerSub!: Subscription;
+
+  constructor(private router: Router) {
+    this.routerSub = this.router.events.subscribe(event => {
+      if (event instanceof NavigationEnd) {
```
Comment (Phantomus) [inline src/app/core/components/navigation/navigation.component.ts]: better to use `.pipe(filter)` to check NavigationEnd

File: src/app/core/components/navigation/navigation.component.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<div>
+  <button [ngClass]="{'active': activeTab === activeRouteNavigationEnum.CharactersRoute}" (click)="toCharacters()">Characters</button> /
```
Comment (Phantomus) [inline src/app/core/components/navigation/navigation.component.html]: also you can do the same with

`[class.active]="activeTab === activeRouteNavigationEnum.CharactersRoute"`

File: src/app/features/characters/pages/character/character-details.component.html [hunk 5]
```diff
@@ -1,12 +1,13 @@
-
 <div class="wrapper">
+  <app-banner></app-banner>
+
   <div class="character">
     <div class="picture-of-character">
-      <img src="{{character?.thumbnailUrl}}" alt="picture-of-character">
+      <img src="{{character.thumbnailUrl}}" alt="picture-of-character">
```
Comment (Phantomus) [inline src/app/features/characters/pages/character/character-details.component.html]: why did you remove `?` ?

File: src/app/features/comics/comics.component.ts [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  imports: [ ComicCardComponent, BannerComponent],
+  templateUrl: './comics.component.html',
+  styleUrl: './comics.component.scss'
+})
+export class ComicsComponent {
+  comicsList: CustomMarvelComic[] = [];
+  amountOfReceivedComics: number = 0;
+
+  comicsLimit: number = 15;
+  comicsOffset: number = 150;
```
Comment (Phantomus) [inline src/app/features/comics/comics.component.ts:21]: why 150?

File: src/app/core/components/navigation/navigation.component.ts [hunk 7]
```diff
@@ -0,0 +1,58 @@
+import { CommonModule } from '@angular/common';
```
Comment (Phantomus) [inline src/app/core/components/navigation/navigation.component.ts]: you can import only NgClass

File: src/app/features/comics/services/comics-marvel.service.ts [hunk 8]
```diff
@@ -0,0 +1,49 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { SpinnerService } from '@core/services/spinner.service';
+import { map, Observable, tap } from 'rxjs';
+import { CustomMarvelComic, MarvelApiComicsResponse, MarvelComic } from '../models/comics.model';
+
+@Injectable({
+  providedIn: 'root'
```
Comment (Phantomus) [inline src/app/features/comics/services/comics-marvel.service.ts:8]: why do you need it `providedIn: 'root'`?

File: src/app/core/components/navigation/navigation.component.scss (hunk 1 above)
Comment (Ellyia) [reply src/app/core/components/navigation/navigation.component.scss]: I've removed them

File: src/app/core/enums/active-route-nav.enum.ts (hunk 2 above)
Comment (Ellyia) [reply src/app/core/enums/active-route-nav.enum.ts]: I've changed them

File: src/app/core/components/navigation/navigation.component.ts (hunk 3 above)
Comment (Ellyia) [reply src/app/core/components/navigation/navigation.component.ts]: changed

File: src/app/features/comics/comics.component.ts (hunk 6 above)
Comment (Ellyia) [reply src/app/features/comics/comics.component.ts:21]: I just want to see different comics, so I just change the number sometimes)

File: src/app/features/comics/services/comics-marvel.service.ts (hunk 8 above)
Comment (Ellyia) [reply src/app/features/comics/services/comics-marvel.service.ts:8]: agreed seems I don't need it in data service

File: src/app/features/characters/pages/character/character-details.component.html (hunk 5 above)
Comment (Ellyia) [reply src/app/features/characters/pages/character/character-details.component.html]: We've already discussed this, I've added "?"

**Review by Phantomus — APPROVED** [review]
